descriptors/ModuleDescriptor-template.json

```diff
@@ -55,6 +55,28 @@
         }
       ]
     },
+    {
+      "id": "pubsub-subscribers",
```

Comment:
It's not a separate interface in current implementation, to do so you should define these endpoints in a different raml file with different root